.circleci/config.yml

```diff
@@ -98,6 +98,17 @@ commands:
             sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
             rm google-chrome.deb
 
+  save-screencaps:
+    steps:
+      - run:
+          name: Saving screenshots & videos from Cypress
+          command: |
+            mkdir /tmp/artifacts;
+            cp -r ./client/cypress/screenshots ./client/cypress/videos /tmp/artifacts;
+
+      - store_artifacts:
+          path: /tmp/artifacts
```

Comment:
any reason the files need to be copied to `/tmp/artifacts`? could this just be:

```
- store_artifacts:
    path: ./client/cypress/screenshots
- store_artifacts:
    path: ./client/cypress/videos
```

if so, i would probably just put this directly into the steps of the cypress job below

Reply:
I was wanting to name it, so we'd see the step in the process, but that's probably not necessary.